packages/python/plotly/plotly/express/_core.py

```diff
@@ -116,6 +116,14 @@ def get_label(args, column):
         return column
 
 
+def invert_label(args, column):
+    reversed_labels = {value: key for (key, value) in args["labels"].items()}
```

Comment:
This works because the default for `args["labels"]` is set to `{}` but probably we should use `None` instead, what do you think @nicolaskruchten ?

Reply:
Yes we should stop using `{}` as defaults :)

Here btw there's a potential issue if people use the same label for two things... this is a bad idea for a user to do but... ?

Reply:
I created https://github.com/plotly/plotly.py/issues/2562 to talk about this issue since it already exists before this PR.